libs/ng-strapi-auth/src/lib/core/services/auth/auth.service.ts

```diff
@@ -32,6 +34,13 @@ export class AuthService {
         return this.httpClient.post<any>(`${this.authApiBase}/auth/local`, { identifier: username, password: password })
             .pipe(map(response => {
                 if (response.jwt && response.user && response.user.blocked == false) {
+                    if (!this.allowedRoles.includes(response.user.role.type)) {
+                        throw {
```

Comment:
Don't throw an http error when it's on fronted. Throw a basic error and opens a snack-bar as if it was an error but don't mock one